### ISSUE TITLE
webrtc data channel

### ISSUE THREAD

**Issue by smidm** [AlexxIT/go2rtc#889]
Is there a way to pass arbitrary data through WebRTC? WebRTC supports "data channels", but I haven't find any mention here. If it is not possible, how hard it would be to implement this and where to start hacking. Thanks!

**Comment by AlexxIT** [comment]
It's out of scope of this project. It's possible to send data via data channel, but go2rtc supports only media data.

**Comment by Daavidaviid** [comment]
Maybe you can replace this lib by pion webrtc? I'm interested by the topic too.

**Comment by AlexxIT** [comment]
I don't understand what you talking about. This is not lib. This is application. It using pion library.

**Comment by AlexxIT** [comment]
Yes. If he has some special business logic - he can build application from the scratch using pion lib.

**Comment by smidm** [comment]
We have a bunch of RPis with Realsense cameras and we need to stream both RGB and depth to browser. Initial testing with `go2rtc` was flawless, everything worked out of the box! We would like to pass calibration data with the streams.

**Comment by AlexxIT** [comment]
You can stream depth, for example, via WebSocket from your API endpoint. Or open another WebRTC connection with your soft. Or try to change go2rtc source code for your task.

**Comment by bilalmirza-sudo** [comment]
Hi! I have a very important request. Do you have a project/code where you stream RGBD data over weebRTC? Would be really grateful if you could help. Thanks!!

**Comment by Daavidaviid** [comment]
Personally I was able to use pion/webrtc and ffmpeg to stream audio/video over webrtc while using data channel at the same time.

It was mostly straightforward
